### PR TITLE
Fix leaked connection

### PR DESCRIPTION
**Pull request by dbmeneses**
It seems to have been fixed (can't reproduce anymore). I was only able to reproduce it when the project doesn't exist in the server. It was probably cause by the 404 when fetching the project repositories.

### CODE REVIEW

File: sonar-ws/src/main/java/org/sonarqube/ws/client/LocalWsConnector.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import java.io.ByteArrayInputStream;
+import java.io.IOException;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/LocalWsConnector.java]: MINOR Remove this unused import 'java.io.IOException'. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsResponse.java [hunk 2]
```diff
@@ -57,5 +58,7 @@
   Reader contentReader();
 
   String content();
+  
+  void close();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsResponse.java]: MAJOR Add the "@Override" annotation above this method signature [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1161)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/BaseResponse.java [hunk 3]
```diff
@@ -40,4 +41,8 @@ public WsResponse failIfNotSuccessful() {
   public boolean hasContent() {
     return code() != HTTP_NO_CONTENT;
   }
+  
+  public void close() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/BaseResponse.java]: MAJOR Add the "@Override" annotation above this method signature [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1161)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 2 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by henryju** [comment]
LGTM

**Pull request closed without merging**